rts-wildfly/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Use latest jboss/wildfly image as the base
+FROM jboss/wildfly
+
+# Make standalone-rts.xml configuration default
+RUN cp /opt/jboss/wildfly/docs/examples/configs/standalone-rts.xml /opt/jboss/wildfly/standalone/configuration/standalone.xml
+
+# Set the default command to run on boot
+# This will boot WildFly in the standalone rts mode and bind to all interfaces
+CMD ["/opt/jboss/wildfly/bin/standalone.sh", "-b", "0.0.0.0"]
```

Comment:
You do not need to specify the `CMD` again, it'll be reused from the `jboss/wildfly` image.